Digipost.Signature.Api.Client.Direct/DirectJob.cs

```diff
@@ -1,6 +1,20 @@
-﻿namespace Digipost.Signature.Api.Client.Direct
+﻿using Digipost.Signature.Api.Client.Core;
+
+namespace Digipost.Signature.Api.Client.Direct
 {
-    class DirectJob
+    public class DirectJob
     {
+        public string Id { get;}
```

Comment:
This field should probably be renamed to `reference` or similar. We changed it's name from `id` to `reference` in the XSD quite recently (https://github.com/digipost/signature-api-specification/blob/master/schema/xsd/direct-signature-job.xsd#L27)

The ID is generated by the server and will be a part of the response when creating a direct job, therefor the sender won't know the value before retrieving a `DirectJobResponse`.